cloud/azure/bin/key-vault-ses

```diff
@@ -0,0 +1,31 @@
+#! /usr/bin/env bash
+
+source bin/lib.sh
+source ../aws/bin/lib.sh
```

Comment:
suggestion (blocking): I think the relative paths are are relative from wherever the `pwd` is, which is different depending on whether someone is running this from the `civiform` or `civiform-deploy` repo. using `BASH_SOURCE` can avoid this problem e.g. https://github.com/seattle-uat/civiform/blob/main/cloud/shared/bin/deploy#L5-L6